samples/preconsent-mdl/src/main/java/com/android/identity/preconsent_mdl/MainActivity.kt

```diff
@@ -141,11 +143,12 @@ class MainActivity : ComponentActivity() {
 
         // Create three credentials and certify them
         for (n in 0..2) {
-            val pendingCredential = document.createCredential(
+            val pendingCredential = MdocCredential.create(
                 AUTH_KEY_DOMAIN,
                 transferHelper.androidKeystoreSecureArea,
                 CreateKeySettings("".toByteArray()),
-                null
+                null,
+                document
```

Comment:
I don't think this actually works since `create()` is protected. Use the constructor and then Document.addCredential() instead.

Reply:
good point - I didn't check the samples to make sure it all works, but will before the next push